### ISSUE TITLE
Specific scheduling for tasks

### ISSUE THREAD

**Issue by SamuelDodee** [mikeyaworski/ABS#55]
I see more _sophisticated_ tasks (this/that quizzes, quizzes and surveys) could interfere with searches. They use its specific tab, stopping searches.
Increasing the number of searches is my workaround and I know you love to keep it simple, but a specific scheduling for tasks would be more elegant and useful in case of weak connections.

**Comment by mikeyaworski** [comment]
I don't understand what you mean by that stopping searches. Are you referring to when the schedule starts searching in a tab, and then opens all tasks in separate tabs as well? That's what I do and I'm not sure how that would stop searches in the first tab.

Though I can still consider having a separate schedule for tasks, I don't understand the issue exactly.

**Comment by SamuelDodee** [comment]
After ABS opens all tasks in tabs, those _sophisticated_ ones could use the tab ABS opened for searches, stopping them for a while. You see quizzes "working" in searches tab! Very strange, I know...
I notice the same thing with simple tasks, but it's not a problem because those tasks are already rewarded.

**Comment by mikeyaworski** [comment]
Right, I have noticed that tasks may appear in all tabs, though I wasn't under the impression that they would stop searches. I've never taken a very close look, but I assumed the tasks would pop up on your next search. I still don't see how searches are really stopped.

Just to be clear, you don't mean that your search counter goes from like 50 to 0 all of a sudden, right?

But I think this is a good feature regardless, so I should implement this when I get some time to work on ABS in the future.

**Comment by SamuelDodee** [comment]
> Just to be clear, you don't mean that your search counter goes from like 50 to 0 all of a sudden, right?

Right!
ABS search counter is counting 1 by 1 anyway, but search tab is working with a task. When that task ends, search resumes.
I.e.
ABS starts searches and it's opening tasks
50
49
Tasks start
48
47
A task "stops" searches
46
45
44
43
42
Task ends and search resumes
41
40
...

**Comment by mikeyaworski** [comment]
Now that you mention it, I think I have seen this behaviour before. Thanks for bringing this up!
